packages/datatrak-web/src/views/AccountSettingsPage/ChangePasswordSection/ChangePasswordForm.tsx

```diff
@@ -95,7 +94,7 @@ export const ChangePasswordForm = () => {
   const submissionShouldBeDisabled = isValidating || !isValid || isSubmitting;
 
   return (
-    <StyledForm onSubmit={handleSubmit(attemptPasswordChange)} formContext={formContext}>
```

Comment:
Is this change intentional? Seems like it will break the change password form?

Reply:
Yep - that reusable component in `ui-components` does the wrapping of the `onSubmit` handler for us